config/initializers/stripe.rb

```diff
@@ -4,3 +4,8 @@
 }
 
 Stripe.api_key = Rails.configuration.stripe[:secret_key]
+
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded', CreatePayment.new
```

Comment:
On second thought I think it's a bit overkill for these Stripe event handlers to be in the services directory. Maybe these two handlers should be moved to a StripeHandler module with inline classes handling the specific event type.